cdk/lib/mobile-save-for-later.ts

```diff
@@ -99,7 +99,13 @@ export class MobileSaveForLater extends GuStack {
     const saveForLaterApi = new GuApiGatewayWithLambdaByPath(this, {
       app,
       restApiName: `${app}-api-${this.stage}`,
-      monitoringConfiguration: { noMonitoring: true }, //TODO: configure monitoring (this is not setup in current CFN)
+      monitoringConfiguration: {
```

Comment:
You might want to set this up as a prop in `MobileSaveForLaterProps`. This would allow you to pass in this 5XX alarm configuration in `PROD` and keep `{ noMonitoring: true }` (which would be closer to what you had before with the throttling alarms IIUC)

Reply:
Yes, great idea. 👍